views/ViewChart.react.js

```diff
@@ -1,24 +1,49 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { connect } from 'react-redux'
 import { compose } from 'redux'
 import { withStyles } from '@material-ui/core/styles'
-
-import { Typography } from '@material-ui/core'
+import { Button, Drawer, Typography } from '@material-ui/core'
+import FilterListIcon from '@material-ui/icons/FilterList'
 import { chartStyles } from './styles/chart_styles'
 import AppLayout from './layouts/AppLayout'
 import BarGraph from './components/BarGraph'
 import GraphTable from './components/GraphTable'
+import { routes } from './routes/routes'
+import ChartFilterForm from './forms/CharFilterForm'
 
 const ViewChart = ({ graph, classes }) => {
-  const { title, description } = graph
+  const { title, description, filters } = graph
+  const [showFilterMenu, setFilterMenuToggle] = useState(false)
+  const openFilterMenu = () => setFilterMenuToggle(true)
+  const onCloseFilterMenu = () => setFilterMenuToggle(false)
+  const handleSubmit = (updatedFilters) =>
+    window.location.assign(routes.chart(graph.chart_id, updatedFilters))
```

Comment:
we should nest the query params for filters in a `filters` object.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/248/commits/147bf8a9965b532f34bb176e1e651f339c9bf387